app/Models/User.php

```diff
@@ -26,6 +26,16 @@ class User extends Authenticatable
         'is_admin',
         'is_active'
     ];
+    
+    protected $guarded = [
+        'is_admin',
+        'is_active'
```

Comment:
```suggestion
        'is_active',
```
phần tử cuối của mảng trên nhiều dòng phải có dấu phẩy